scripts/fast-reboot

```diff
@@ -277,8 +277,13 @@ function backup_database()
 function setup_control_plane_assistant()
 {
     if [[ -n "${ASSISTANT_IP_LIST}" && -x ${ASSISTANT_SCRIPT} ]]; then
-        debug "Setting up control plane assistant: ${ASSISTANT_IP_LIST} ..."
-        ${ASSISTANT_SCRIPT} -s ${ASSISTANT_IP_LIST} -m set
+        # TH3 HW is not capable of VxLAN programming thus skipping TH3 platforms
+        if [[ "$HWSKU" != "DellEMC-Z9332f-M-O16C64" && "$HWSKU" != "DellEMC-Z9332f-M-O16C64-lab" ]]; then
```

Comment:
`$HWSKU` can be replaced by `${HWSKU}` for consistency and probably bash safety too.